src/test/scala/is/hail/io/ImportGDBSuite.scala

```diff
@@ -0,0 +1,140 @@
+package is.hail.io
+
+import is.hail.SparkSuite
+import is.hail.expr.TStruct
+import is.hail.io.vcf.{GenericRecordReader, LoadGDB, LoadVCF}
+import org.apache.spark.sql.catalyst.expressions.GenericRow
+import org.testng.annotations.Test
+
+class ImportGDBSuite extends SparkSuite {
+
+  val loader = "src/test/resources/sample2loader.json"
+  val query = "src/test/resources/sample2query.json"
+  val workspace = "src/test/resources/tdbworkspace"
+  val arrName = "sample2Array"
+  val ref = "/Users/jgoldsmi/build/Homo_sapiens_assembly19.fasta"
+  val reader = new GenericRecordReader(Set.empty)
+  val gdbVariantSampleMatrix = LoadGDB(hc, reader, loader, query, workspace, arrName, ref)
+  val vcfVariantSampleMatrix = LoadVCF(hc, reader, "src/test/resources/sample2.vcf")
```

Comment:
Generally, tests should be self-contained so that individual ones can be run. It also helps debugging them when we know that there is no possibility of shared state between tests.

Let's copy these into the test methods that use them.